docs_src/V3TopLevelMigration.md

```diff
@@ -111,6 +111,11 @@ The following sections describe what you need to be aware for the different serv
 
 The Event/Reading data stored by Core Data is considered transient and of little value once it has become old. The V3 versions of these data collections have minimal changes from their V2 counter parts. 
 
+#### API Change
+- Add Event
+    POST endpoint is now  `/event/{serviceName}/{profileName}/{deviceName}/{sourceName}`. This is so that the metadata contains the device service which generated the new event
```

Comment:
> This is so that the metadata contains the device service which generated the new event

is it a correct sentence?  do you understand what it means?